test/unit/uk/gov/hmrc/exports/models/ead/parsers/MrnStatusParserTestData.scala

```diff
@@ -31,7 +31,7 @@ object MrnStatusParserTestData {
     <p:DeclarationStatusDetails>
       <p:Declaration>
         <p:AcceptanceDateTime>
-          <p1:DateTimeString formatCode="304">20190702110757Z</p1:DateTimeString>
+          <p1:DateTimeString formatCode="304">20190102110757Z</p1:DateTimeString>
```

Comment:
You change the date to be in the summer?

Reply:
Winter, even.   So 11:07 in winter must be displayed as 11:07?
What should that time be in summer?  Can we have a test for that?

Reply:
to be in Winter actually. I am changing from July to January, in order to provide an example of a time that should not change.
```
old: 20190702110757Z
new: 20190102110757Z
```

Reply:
Gotcha!

Reply:
The test has been updated to reflect that: check `MrnStatusParserSpec.scala`

```
 mrnStatus.acceptanceDateTime mustBe Some("02 January 2019 at 11:07am")
 mrnStatus.receivedDateTime mustBe "02 July 2019 at 12:08pm"
 mrnStatus.releasedDateTime mustBe Some("02 July 2019 at 12:09pm")
```

Reply:
Yes, I get it now.  Merge away.